packages/containers/src/DeleteResource/DeleteResource.container.js

```diff
@@ -106,6 +106,9 @@ export class DeleteResource extends React.Component {
 				cancelAction={cancelAction}
 				validateAction={validateAction}
 				getComponent={this.props.getComponent}
+				onHide={() =>
+					this.props.dispatchActionCreator('DeleteResource#cancel', event, { model: resourceInfo })
```

Comment:
To avoid passing a new function ref at each render, can you make it a class function ?